adafruit_rgbled.py

```diff
@@ -109,28 +114,32 @@ def __init__(self, red_pin, green_pin, blue_pin, invert_pwm=False):
         self._current_color = (0, 0, 0)
         self.color = self._current_color
 
-    def __enter__(self):
+    def __enter__(self) -> "RGBLED":
         return self
 
-    def __exit__(self, exception_type, exception_value, traceback):
+    def __exit__(self, exception_type, exception_value, traceback) -> None:
```

Comment:
You can find the types for the arguments of `__exit__()` [here](https://github.com/adafruit/Adafruit_CircuitPython_Motor/blob/9c3de3abce00b50ba936f4f8daad0a8a6bee34a6/adafruit_motor/servo.py#L164-L170).

Reply:
Thanks! That's done. I'll use that in a couple of other libraries that I've added typing to.